src/firefly/js/ui/DropDownContainer.jsx

```diff
@@ -84,20 +86,25 @@ export class DropDownContainer extends Component {
     }
 
     render() {
-        const {footer} = this.props;
+        const {footer, alerts} = this.props;
         var { visible, selected }= this.state;
         var view = dropDownMap[selected];
 
         if (!visible) return <div/>;
         return (
-            <div className='DD-ToolBar'>
-                <div className='DD-ToolBar__content'>
-                    {view}
-                </div>
-                <div id='footer' className='DD-ToolBar__footer'>
-                    {footer}
-                    <div className='DD-ToolBar__version'>
-                        {getVersion()}
+            <div>
+                <div className='DD-ToolBar'>
+                    {alerts || <Alerts />}
```

Comment:
alerts from property is meant to replace the default <Alerts> panel.  that way, one can change its look and feel.